jsk_2016_01_baxter_apc/node_scripts/helper.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
This is python module and not a node script, right?
You can put this as python module with `catkin_setup_python()` in cmake and setup.py file.